### PR TITLE
Content changes

### PR DESCRIPTION
**Pull request by lfdebrux**
https://trello.com/c/PBTdbwko/40-2-esignature-prototype-content-changes-reqd-before-sign-off

Changes have been pushed to Heroku so can be previewed there (see [comment on ticket](https://trello.com/c/PBTdbwko/40-2-esignature-prototype-content-changes-reqd-before-sign-off#comment-5ea155e0c6c1d05735591616) for details).

### CODE REVIEW

File: app/views/framework-countersigned.html [hunk 1]
```diff
@@ -63,16 +64,12 @@ <h2 class="govuk-heading-m">
...
-      </p>
-
       <p>
       <a class="govuk-link" href="#">View your signed Framework Agreement</a>
       </p>
 
       <p>
-      You can start selling your G-Cloud 12 services on the Digital Marketplace from Monday 28 September 2025.
+      You can start selling your G-Cloud 12 services on the Digital Marketplace from 28 September 2020.
       </p>
```
Comment (NoraGDS) [inline app/views/framework-countersigned.html:73]: Please can we add 'Monday' back in?

File: app/views/email-02.html [hunk 2]
```diff
@@ -27,11 +27,15 @@ <h2 class="govuk-heading-m">
...
       </p>
 
       <p>
-      You'll need to wait for CCS to countersign your agreement before you can sell services.
+      We'll email you when the countersigned agreement is available (this is normally within 3 days).
+      </p>
+
+      <p>
+      You'll need to wait for CCS to countersign your agreement before you can sell services from 28 September 2020.
       </p>
```
Comment (NoraGDS) [inline app/views/email-02.html:39]: Please can we amend this:

Once you have a countersigned agreement, you can sell services from Monday 28 September 2020.

**Review by NoraGDS — COMMENTED** [review]
Thank you for updating these.

File: app/views/email-02.html (hunk 2 above)
Comment (lfdebrux) [reply app/views/email-02.html:39]: Done

File: app/views/framework-countersigned.html (hunk 1 above)
Comment (lfdebrux) [reply app/views/framework-countersigned.html:73]: Done

**Review by NoraGDS — COMMENTED** [review]
thank you.

**Pull request merged**